src/Transport/Addressing/Partitioning/ReadOnlySettingsExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace NServiceBus.AzureServiceBus.Addressing.Partitioning
+{
+    using System.Collections.Generic;
+    using System.Configuration;
+    using System.Linq;
+    using Settings;
+    using Transport.AzureServiceBus;
+
+    public static class ReadOnlySettingsExtensions
+    {
+        public static List<RuntimeNamespaceInfo> GetPartitioningNamespaces(this ReadOnlySettings settings)
```

Comment:
Looking at the sample above, why do we even need this strategy? All it does is returning the found `RuntimeNamespaceInfo` entities, doesn't it?